test/features/environment.py

```diff
@@ -68,12 +69,27 @@ def after_all(context):
     #        os.environ[k] = v
 
 
+def before_step(context, _):
+    context.atprompt = False
+
+
 def after_scenario(context, _):
     """Cleans up after each test complete."""
 
     if hasattr(context, 'cli') and not context.exit_sent:
-        # Terminate nicely.
-        context.cli.terminate()
+        # Quit nicely.
+        if not context.atprompt:
```

Comment:
I'm wondering -- why are we expecting a certain output after scenarios have already been run? Wouldn't it be more explicit if the scenario ended with `and we wait for prompt` since that step does the same thing as this code?

Then we can drop the `atprompt` piece of the context and just rely on the scenario itself.